cmake/Helper.cmake

```diff
@@ -189,6 +189,17 @@ function(wasmedge_add_library target)
     set_target_properties(${target} PROPERTIES
       INSTALL_RPATH "$ORIGIN"
     )
+  elseif(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+    # Set properties to build as a framework on macOS and iOS
+    set_target_properties(${target} PROPERTIES
+      FRAMEWORK TRUE
+      FRAMEWORK_VERSION C
+      MACOSX_FRAMEWORK_IDENTIFIER "com.wasmedge.${target}"
```

Comment:
It should be `org.wasmedge...`. WasmEdge itself is not a company.

Reply:
got it, changing it.